### ISSUE TITLE
Unable to add SLW-10M pump

### ISSUE THREAD

**Issue by Simonv92** [chrisc123/jebao_aqua-homeassistant#14]
Hi,
I successfully added the pump to the app linked to my account. 
When I load the integration by entering my account details it doesn't detect any devices
What can it be? 
Thank you
Simone

**Comment by Simonv92** [comment]
I've tryed again. On the app I have control of my pump, oh Home Assistant it doesn't show any pump. 

![Image](https://github.com/user-attachments/assets/dc7286fe-d778-4bba-966d-4945bd65d649)

I already tryed to delete the integration and reload but nothing happen 

On my phone I have this:

![Image](https://github.com/user-attachments/assets/64ad0e38-8a2b-40b9-a519-1ff4f8485d3c)

**Comment by chrisc123** [comment]
Hi - did you use HACS to install the integration? If you try downloading the prerelease version 0.3 hopefully it will work for you:

<img width="614" alt="Image" src="https://github.com/user-attachments/assets/8a4d9346-1bc1-48e2-93d4-c40347dcf0a6" />

<img width="281" alt="Image" src="https://github.com/user-attachments/assets/d338470f-02ae-4369-bcb9-d7b7015a5ab8" />

**Comment by Simonv92** [comment]
Yes, I've downloaded the Jeabo Aqua integration via HACS some times ago. But now if I go in HACS I cannot find it anymore... 

![Image](https://github.com/user-attachments/assets/9e6bd91f-47cc-4e50-9509-b84a70f1685f)

**Comment by chrisc123** [comment]
Odd - is the repository still configured as a custom repository in HACS?

**Comment by Simonv92** [comment]
No, I've also tryed to setup another HA container, installed HACS but I cannot find the Jebao Aqua integrations... It seems that it's missing on HACS

edit: maybe I've found the issue.. the URL was not set anymore in the custom repository...

**Comment by Simonv92** [comment]
Now it works!! :D

**Comment by chrisc123** [comment]
Great - did it all setup OK? 

This new version is quite a significant change as it does everything locally (rather than using the Jebao cloud) and also uses push based notifications rather than polling the device.

Let me know if you have any problems!

**Comment by Simonv92** [comment]
Thank you again!
I'm doing some basic test, everything seems to work just fine Thank you! :)
-

**Comment by chrisc123** [comment]
[v0.4.1-beta](https://github.com/chrisc123/jebao_aqua-homeassistant/releases/tag/v0.4.1-beta) replaces the cloud-account flow with direct LAN discovery (and the SLW model definition is bundled), so detection no longer depends on your cloud account at all — the pump just needs to be on a network reachable from Home Assistant. Please try the beta and reopen if the SLW-10M still isn't found.

**To install the beta:** HACS → Jebao Aqua → ⋮ → Redownload → enable "Show beta versions" → v0.4.1-beta → restart HA.
